py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -45,6 +48,13 @@ def aot_torch_tensorrt_aten_backend(
         settings=settings,
     )
 
+    logger.debug("Pre-module replacement graph:\n" + str(gm.graph))
```

Comment:
We could put this in pre_aot_module_replacement so that every time it is called, we log both pre and post

Reply:
Moved the debug prints into the `pre_aot_module_replacement ` function